hass_nabucasa/api.py

```diff
@@ -174,6 +175,12 @@ def _do_log_response(
             return
         isok = resp.status < 400
         target = resp.url.path if resp.url.host == self.hostname else ""
+        if len(resp.url.query) > 0:
+            query_params = [
+                f"{key}=***" if value not in ["true", "false"] else f"{key}={value}"
```

Comment:
```suggestion
                f"{key}=***" if value.lower() not in {"true", "false"} else f"{key}={value}"
```
It should not be needed, but you never know.
Also the `{"true", "false"}` set should be defined outside of the loop